components/builder-notify/habitat/config/config.toml

```diff
@@ -1,2 +1,10 @@
 [eventbus]
 {{toToml cfg.eventbus}}
+{{~#if cfg.webhooks}}
+[hub]
+{{~#each cfg.webhooks as |webhook|}}
```

Comment:
could there be multiple `webhooks`? If so, it seems like you would end up with duplicate `[[hub.webhooks]]` tables.

Reply:
Yes, there can be more than one webhook! I would say it is an array of objects, Is there any other way to configure more than one table.

Reply:
oh nevermind. This is correct. I forgot about the double btacket toml table array.